.github/workflows/publish_logs.yml

```diff
@@ -0,0 +1,50 @@
+name: Collect, count and publish logs
```

Comment:
Isn't this a duplicate of `.github/actions/publish-logs/action.yml`?

Reply:
I mean, could you just do `uses: ./.github/actions/publish-logs` here to avoid code duplication?

Reply:
My bad, I duplicated code, will remove it :)

Reply:
@milan-zededa done